Subject: Provide a sensible INSTALLDIRS default for modules installed from CPAN.

Some modules which are included in core set INSTALLDIRS => 'perl'
explicitly in Makefile.PL or Build.PL.  This makes sense for the normal @INC
ordering, but not ours.


---
 lib/CPAN/FirstTime.pm |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/CPAN/FirstTime.pm b/lib/CPAN/FirstTime.pm
index 50bebc3..0136fef 100644
--- a/lib/CPAN/FirstTime.pm
+++ b/lib/CPAN/FirstTime.pm
@@ -952,7 +952,7 @@ sub init {
     my_prompt_loop(prefer_installer => 'MB', $matcher, 'MB|EUMM|RAND');
 
     if (!$matcher or 'makepl_arg make_arg' =~ /$matcher/) {
-        my_dflt_prompt(makepl_arg => "", $matcher);
+        my_dflt_prompt(makepl_arg => "INSTALLDIRS=site", $matcher);
         my_dflt_prompt(make_arg => "", $matcher);
         if ( $CPAN::Config->{makepl_arg} =~ /LIBS=|INC=/ ) {
             $CPAN::Frontend->mywarn( 
@@ -974,7 +974,7 @@ sub init {
     my_dflt_prompt(make_install_arg => $CPAN::Config->{make_arg} || "",
                    $matcher);
 
-    my_dflt_prompt(mbuildpl_arg => "", $matcher);
+    my_dflt_prompt(mbuildpl_arg => "--installdirs site", $matcher);
     my_dflt_prompt(mbuild_arg => "", $matcher);
 
     if (exists $CPAN::HandleConfig::keys{mbuild_install_build_command}
-- 
tg: (daf8b46..) debian/cpan_definstalldirs (depends on: upstream)
